### ISSUE TITLE
Admob real ad not showing the ads.

### ISSUE THREAD

**Issue by ravanxgames** [Shin-NiL/Godot-Android-Admob-Plugin#70]
**Godot version:**
3.2.2
**AdMob Plugin version:**
4.0.0
**Issue description:**

is_real = true for production build done. But the real ad not showing in the game. When i give the test Ad appID its working perfectly. 

How to solve the issue? no error showing. while taking signed build, i just followed all setting.

**Comment by M0liusX** [comment]
@Lamelynx I haven't tried it since then... sorry. However, I've only ran it using a debug apk. I am under the impression that it needs to be a release version of the apk to work. If you ever figure out the real reason let me know too. Haha

**Comment by Lamelynx** [comment]
@M0liusX 
@Shin-NiL 

I have released open beta but only got test ads.
I have set AndroidManifest.xml
`<meta-data
            android:name="com.google.android.gms.ads.APPLICATION_ID"
            android:value="ca-app-pub-XXXXXXXXXXXXXX~XXXXXXXX"/>`

![admob](https://user-images.githubusercontent.com/43991289/92093259-6ebd3f00-edd3-11ea-9b53-35c5f630a1e3.png)

Set permission "Access Network State" 

I found out i missed the "internet" permission so it may be this that cause the problem?

Anyway, test ads works perfectly (with my personal admob id's) so the code would probably not be the problem.

**Comment by Shin-NiL** [comment]
@Lamelynx are you getting error code 3 too? If so, check [this](https://support.google.com/admob/thread/3494603).

**Comment by Lamelynx** [comment]
@Shin-NiL 

When i added "Internet" permission I did get error code 3 so it seems to be fine.
As I understand it there is no ads to load, hopefully everything else is ok.
Thanks for the help.

**Comment by thermocoffee** [comment]
Sorry to hijack this thread but it's related to my problem. I'm done with the silly game I created. I just put in 1 interstitial ad at the end of the game.  It works great when I use googles testings id's. But when I put my id's in, it still shows "test ad". Is this normal behaviour or should I be seeing real ads?

**Comment by Lamelynx** [comment]
You probably won't see any real ads. You should get error code 3 in logcat. After you checked "is real" I think you need to save your project before export. I also believe that you can't get any real ads if exporting with debug.
Have you dubble checked AndroidManifest.xml?

**Comment by albemala** [comment]
I was receiving the same error code 3, I solved the problem following this answer on SO: https://stackoverflow.com/a/57964366/3305693

TL;DR: 
- set is_real = true
- publish a release build of your app on the Google Play Store (you can publish it as Open testing / Beta)
- you should then start seeing real ads (make sure to install the app you just published from the store, not from a local APK)

**Comment by disanapa** [comment]
Version 4.1.1
Doesn't show real ads, only test ads.
